Sources/Sentry/SentryEnvelope.m

```diff
@@ -104,7 +104,7 @@ - (instancetype)initWithEvent:(SentryEvent *)event
         event.context = nil;
         event.sdk = nil;
         error = nil;
-        json = [SentrySerialization dataWithJSONObject:[event serialize] error:&error];
+        [SentrySerialization dataWithJSONObject:[event serialize] error:&error];
```

Comment:
l: Is this right? You create a NSData but don't use anywhere?

Reply:
Yes, if you check the code below it makes sense.